CHANGELOG.md

```diff
@@ -1,5 +1,88 @@
+* Fix copyright header for some files ([#1371](https://github.com/kubeflow/tf-operator/pull/1371), @DeliangFan)
+* fix incorrect torch env population ([#1361](https://github.com/kubeflow/tf-operator/pull/1361), @Jeffwan)
+* fix: Resolve scheme registration issue for defaulters ([#1360](https://github.com/kubeflow/tf-operator/pull/1360), @Jeffwan)
+* Fix XGBoost container name in log message ([#1362](https://github.com/kubeflow/tf-operator/pull/1362), @andreyvelich)
+* Fix postsubmit job using PULL_BASE_SHA ([#1344](https://github.com/kubeflow/tf-operator/pull/1344), @Jeffwan)
+* Fix all client request that needs contexts ([#1292](https://github.com/kubeflow/tf-operator/pull/1292), @Jeffwan)
+
+## Misc
+
+* Update manifests with  latest image tag [#1406](https://github.com/kubeflow/tf-operator/pull/1406) ([johnugeorge](https://github.com/johnugeorge))
+* Add simple verification jobs ([#1391](https://github.com/kubeflow/tf-operator/pull/1391), @Jeffwan)
+* chore: Bump kubeflow/common version to 0.3.7 ([#1388](https://github.com/kubeflow/tf-operator/pull/1388), @Jeffwan)
+* chore(doc): Update README.md ([#1387](https://github.com/kubeflow/tf-operator/pull/1387), @Jeffwan)
+* Remove lagacy tf-operator from the codebase ([#1378](https://github.com/kubeflow/tf-operator/pull/1378), @thunderboltsid)
+* chore: Update manifest image tag ([#1364](https://github.com/kubeflow/tf-operator/pull/1364), @Jeffwan)
+* add example for mxnet and pytorch ([#1373](https://github.com/kubeflow/tf-operator/pull/1373), @DeliangFan)
+* docs: Update to use kubectl kustomize in instructions ([#1356](https://github.com/kubeflow/tf-operator/pull/1356), @terrytangyuan)
+* Clean up manifests and remove unused files ([#1349](https://github.com/kubeflow/tf-operator/pull/1349), @Jeffwan)
+* 1322: Modified manifests to use all-in-one training-operator ([#1346](https://github.com/kubeflow/tf-operator/pull/1346), @deepak-muley)
+* chore: Update changelog.md ([#1339](https://github.com/kubeflow/tf-operator/pull/1339), @Jeffwan)
+* Move root level docs to docs folder ([#1338](https://github.com/kubeflow/tf-operator/pull/1338), @Jeffwan)
+* Temporarily add Jeffwan@ to OWNERS ([#1287](https://github.com/kubeflow/tf-operator/pull/1287), @Jeffwan)
+
+## Testing
+
+* Update include dirs in prow config ([#1374](https://github.com/kubeflow/tf-operator/pull/1374), @andreyvelich)
+* Enable e2e test against universal operator  ([#1336](https://github.com/kubeflow/tf-operator/pull/1336), @Jeffwan)
+* Use PULL_PULL_SHA for image ([#1334](https://github.com/kubeflow/tf-operator/pull/1334), @PatrickXYS)
+
+
+## [v1.2.1](https://github.com/kubeflow/tf-operator/tree/v1.2.1) (2021-08-27)
+
+[Full Changelog](https://github.com/kubeflow/tf-operator/compare/v1.3.0-alpha.2...v1.2.1)
+
+**Closed issues:**
+
+- volcano scheduler with customized queue [\#1377](https://github.com/kubeflow/tf-operator/issues/1377)
```

Comment:
Where is the redundant `\` from?

Reply:
I think this is probably from different release scripts. I will check without @gaocegege on this.  This is the latest one I use.